src/record_syscall.cc

```diff
@@ -2004,6 +2005,14 @@ static Switchable prepare_ioctl(RecordTask* t,
                                       args.wLength);
       return PREVENT_SWITCH;
     }
+    case IOCTL_MASK_SIZE(FS_IOC_FIEMAP): {
+      auto argsp = syscall_state.reg_parameter<typename Arch::fiemap>(3, IN);
```

Comment:
Make this `auto argsp = remote_ptr<typename Arch::fiemap>(regs.arg3());`. No need to use `reg_parameter` here and it's probably best to avoid using `reg_parameter` twice on the same register.

Reply:
Done. For `regs` is something like `t->regs()` ok?